### PR TITLE
ASO optimizations

### PR DESCRIPTION
**Pull request by ncoelho**
Store description in English and Portuguese BR. More keywords relevant for search on all languages. Portuguese PT translation added to description and keywords.

### CODE REVIEW

File: fastlane/metadata/en-US/keywords.txt [hunk 1]
```diff
@@ -1 +1 @@
-bitcoin, segwit, replace by fee, blockchain, btc, plausible deniability, cryptocurrency, wallet
+bitcoin, bitcoin wallet, segwit, buy bitcoin, sell bitcoin, crypto, btc, replace by fee, blockchain, btc, plausible deniability, cryptocurrency, wallet, lightning network, bread, brd, samurai, bitcoin.com, bitcoin.org, ripple, xrp, ethereum, eos, neo, cardano, stellar, litecoin, monero, tron, sentinel, cold storage, price, euro, dollar, digital
```
Comment (Overtorment) [inline fastlane/metadata/en-US/keywords.txt]: wow, easy, cowboy. keywords can only have 100 chars

File: fastlane/metadata/es-ES/keywords.txt [hunk 2]
```diff
@@ -1 +1 @@
-bitcoin, segwit, wallet, blockchain, btc, plausible deniability, cryptocurrency, crypto
\ No newline at end of file
+bitcoin, bitcoin wallet, segwit, buy bitcoin, sell bitcoin, crypto, btc, replace by fee, blockchain, btc, plausible deniability, cryptocurrency, wallet, lightning network, bread, brd, samurai, bitcoin.com, bitcoin.org, ripple, xrp, ethereum, eos, neo, cardano, stellar, litecoin, monero, tron, sentinel, cold storage, price, euro, dollar, digital
```
Comment (Overtorment) [inline fastlane/metadata/es-ES/keywords.txt]: wow, easy, cowboy. keywords can only have 100 chars

File: fastlane/metadata/pt-BR/keywords.txt [hunk 3]
```diff
@@ -1 +1 @@
-bitcoin,segwit,wallet,blockchain,btc,negação plausível,cryptomoeda,crypto,carteira
\ No newline at end of file
+bitcoin, bitcoin wallet, carteira bitcoin, carteira, segwit, buy bitcoin, sell bitcoin, crypto, btc, replace by fee, blockchain, btc, plausible deniability, cryptocurrency, wallet, lightning network, bread, brd, samurai, bitcoin.com, bitcoin.org, ripple, xrp, ethereum, eos, neo, cardano, stellar, litecoin, monero, tron, sentinel, cold storage, price, euro, dollar, digital
```
Comment (Overtorment) [inline fastlane/metadata/pt-BR/keywords.txt]: wow, easy, cowboy. keywords can only have 100 chars

File: fastlane/metadata/pt-PT/keywords.txt [hunk 4]
```diff
@@ -0,0 +1 @@
+bitcoin, bitcoin wallet, carteira bitcoin, carteira, segwit, buy bitcoin, sell bitcoin, crypto, btc, replace by fee, blockchain, btc, plausible deniability, cryptocurrency, wallet, lightning network, bread, brd, samurai, bitcoin.com, bitcoin.org, ripple, xrp, ethereum, eos, neo, cardano, stellar, litecoin, monero, tron, sentinel, cold storage, price, euro, dollar, digital
```
Comment (Overtorment) [inline fastlane/metadata/pt-PT/keywords.txt]: wow, easy, cowboy. keywords can only have 100 chars

File: fastlane/metadata/ru/keywords.txt [hunk 5]
```diff
@@ -1 +1 @@
-биткоин, биткойн, кошелек, блокчейн, криптовалюта, крипта, биток, bitcoin, wallet, blockchain, btc
\ No newline at end of file
+биткоин, биткойн, кошелек, блокчейн, криптовалюта, крипта, биток, bitcoin, bitcoin wallet, segwit, buy bitcoin, sell bitcoin, crypto, btc, replace by fee, blockchain, btc, plausible deniability, cryptocurrency, wallet, lightning network, bread, brd, samurai, bitcoin.com, bitcoin.org, ripple, xrp, ethereum, eos, neo, cardano, stellar, litecoin, monero, tron, sentinel, cold storage, price, euro, dollar, digital
```
Comment (Overtorment) [inline fastlane/metadata/ru/keywords.txt]: wow, easy, cowboy. keywords can only have 100 chars

File: fastlane/metadata/ua/keywords.txt [hunk 6]
```diff
@@ -1 +1 @@
-биткоин, биткоїн, биткоін, биткойн, кошелек, блокчейн, криптовалюта, крипта, bitcoin, segwit, blockchain, btc
+биткоин, биткоїн, биткоін, биткойн, кошелек, блокчейн, криптовалюта, крипта, bitcoin, bitcoin wallet, segwit, buy bitcoin, sell bitcoin, crypto, btc, replace by fee, blockchain, btc, plausible deniability, cryptocurrency, wallet, lightning network, bread, brd, samurai, bitcoin.com, bitcoin.org, ripple, xrp, ethereum, eos, neo, cardano, stellar, litecoin, monero, tron, sentinel, cold storage, price, euro, dollar, digital
```
Comment (Overtorment) [inline fastlane/metadata/ua/keywords.txt]: wow, easy, cowboy. keywords can only have 100 chars

**Comment by ncoelho** [comment]
fixed, didn't know the limit

**Pull request merged**